packages/config/categories.ts

```diff
@@ -4,12 +4,17 @@ export type CategoryInfo = {
     description: string;
 };
 
-export type CategoryKeys = "css" | "tools" | "db" | "markdown";
+export type CategoryKeys = "common" | "css" | "db" | "markdown" | "tools";
 export type CategoryDetails = Record<CategoryKeys, CategoryInfo>;
 
 export type AdderCategories = Record<CategoryKeys, string[]>;
 
 export const categories: CategoryDetails = {
+    common: {
+        id: "common",
+        name: "Common Tooling",
```

Comment:
"Tools" and "Common Tooling" seem too similar as category names. perhaps this category could be something like "Code hygiene"?

Reply:
how about "Linters"?

Reply:
To me, prettier is a formatter, which is a different category than linter. "Code quality" could be another option

Reply:
I like code quality tbh